common/apps/apps.go

```diff
@@ -23,8 +23,23 @@ import (
 	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/appc/spec/schema/types"
 )
 
+// AppImageType describes a type of an image parameter. The parameter
+// can be either a hash, a URL, a path, a name or none of these. The
+// last option means that the application will have to deduce the
```

Comment:
Heh, by "The last option" here I meant the "none of these" in the previous sentence, not the first item in enumeration below.

I'll reword it.